# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * payment_dpo
#
# Translators:
# Wil Odoo, 2025
# Junko Augias, 2025
#
# "Junko Augias (juau)" <juau@odoo.com>, 2025.
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~18.3\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-05-09 20:37+0000\n"
"PO-Revision-Date: 2025-10-16 00:20+0000\n"
"Last-Translator: \"Junko Augias (juau)\" <juau@odoo.com>\n"
"Language-Team: Japanese <https://translate.odoo.com/projects/odoo-s18-3/"
"payment_dpo/ja/>\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Weblate 5.12.2\n"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid ""
"An error occurred during processing of your payment (code %(code)s: "
"%(explanation)s). Please try again."
msgstr "お支払いの処理中にエラーが発生しました(コード%(code)s: %(explanation)s) もう一度お試しください。"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__code
msgid "Code"
msgstr "コード"

#. module: payment_dpo
#: model_terms:ir.ui.view,arch_db:payment_dpo.payment_provider_form
msgid "Company Token"
msgstr "会社トークン"

#. module: payment_dpo
#: model:ir.model.fields.selection,name:payment_dpo.selection__payment_provider__code__dpo
msgid "DPO"
msgstr "DPO"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__dpo_company_token
msgid "DPO Company Token"
msgstr "DPO会社トークン"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__dpo_service_ref
msgid "DPO Service ID"
msgstr "DPOサービスID"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__display_name
#: model:ir.model.fields,field_description:payment_dpo.field_payment_transaction__display_name
msgid "Display Name"
msgstr "表示名"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__id
#: model:ir.model.fields,field_description:payment_dpo.field_payment_transaction__id
msgid "ID"
msgstr "ID"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid "No transaction found matching reference %s."
msgstr "参照に一致する取引が見つかりません%s。"

#. module: payment_dpo
#: model:ir.model,name:payment_dpo.model_payment_provider
msgid "Payment Provider"
msgstr "決済プロバイダー"

#. module: payment_dpo
#: model:ir.model,name:payment_dpo.model_payment_transaction
msgid "Payment Transaction"
msgstr "支払取引"

#. module: payment_dpo
#: model_terms:ir.ui.view,arch_db:payment_dpo.payment_provider_form
msgid "Service ID"
msgstr "サービスID"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_provider.py:0
msgid "The communication with the API failed."
msgstr "APIとのやり取りに失敗しました。"

#. module: payment_dpo
#: model:ir.model.fields,help:payment_dpo.field_payment_provider__code
msgid "The technical code of this payment provider."
msgstr "この決済プロバイダーのテクニカルコード。"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid "Unknown status code: %s"
msgstr "不明なステータスコード: %s"
